app/components/provider_interface/interview_card_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="app-interview-card">
   <div class="app-interview-card__time">
     <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0">
-    <%= link_to provider_interface_application_choice_interviews_path(interview.application_choice), class: "govuk-link govuk-link--no-visited-state" do %>
+    <%= link_to provider_interface_application_choice_interviews_path(interview.application_choice, anchor: "interview-#{interview.id}"), class: "govuk-link govuk-link--no-visited-state" do %>
```

Comment:
Having had another look through existing examples of testing url anchors, we can add the following test to the component specs: 
```
it 'renders the application choice interviews anchored url' do
    expect(render.css('a').attr('href').value).to eq("/provider/applications/{application_choice.id}/interviews#interview-#{interview.id}"
end
```